src/library/export/engineprimeexportjob.cpp

```diff
@@ -204,14 +193,13 @@ void exportMetadata(djinterop::database* pDatabase,
         // number of beats from there until the end of the track in order to
         // correctly assign an index for the last beat.
         const auto lastBeatPlayPos =
-                beats->findPrevBeat(mixxx::audio::FramePos::fromEngineSamplePos(
-                        sampleOffsetToPlayPos(sampleCount)));
+                beats->findPrevBeat(mixxx::audio::FramePos(frameCount));
```

Comment:
`mixxx::audio::kStartFramePose + frameCount`? Doesn't matter but would be the correct calculation.